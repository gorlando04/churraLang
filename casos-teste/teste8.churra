amigo "Felipe" contribui 100
amigo "Marcos" contribui 100
item comida preco_unit 20 quantidade 5
item entretenimento preco_unit 25 quantidade 1
rateio igual