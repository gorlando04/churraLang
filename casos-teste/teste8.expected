Total do churrasco: R$125.00
Felipe deve pagar: R$62.50
Marcos deve pagar: R$62.50